### PR TITLE
`quick-new-issue` - Add icon to dropdown

### LINKED ISSUE #6937: `quick-new-issue` - Icon missing
### Description

This is a new change, we should keep supporting the old icon-less menu

Screenshot 5

### How to replicate the issue + URL

https://github.com/fregante/empty

### Extension version

23.9.19

### Browser(s) used

Chrome

### PR DESCRIPTION
**Pull request by sir-kokabi**
Closes #6937 

## Test URLs
Click the + button in the top-right corner beside search box.

## Screenshot
image

### CODE REVIEW

File: source/features/quick-new-issue.tsx [hunk 1]
```diff
@@ -23,6 +23,12 @@ function add(dropdownMenu: HTMLElement): void {
 		<li role="presentation" aria-hidden="true" data-view-component="true" className="ActionList-sectionDivider"/>,
 		<li data-targets="action-list.items" role="none" data-view-component="true" className="ActionListItem">
 			<a href={buildRepoURL('issues/new/choose')} tabIndex={-1} role="menuitem" data-view-component="true" className="ActionListContent">
+				<span className="ActionListItem-visual ActionListItem-visual--leading">
+					<svg aria-hidden="true" height="16" viewBox="0 0 16 16" version="1.1" width="16" data-view-component="true" className="octicon octicon-repo-push">
```
Comment (fregante) [inline source/features/quick-new-issue.tsx:27]: You can import the icon from the "octagons" package. Refer to its usage in other features

Comment (sir-kokabi) [reply source/features/quick-new-issue.tsx]: It's done.

Comment (fregante) [reply source/features/quick-new-issue.tsx]: * octicons 😅 damn autocorrect

Comment (sir-kokabi) [reply source/features/quick-new-issue.tsx]: and we have a library named [octagon](https://github.com/Tripwire/octagon) which is thankfully unrelated to icons. 😁

**Comment by fregante** [comment]
I updated the readme

Screenshot 1

**Pull request merged**